packages/manager/apps/sap-features-hub/src/components/Form/SelectField.component.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  OdsFormField,
+  OdsSelect,
+  OdsSpinner,
+  OdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  OdsSelectChangeEventDetail,
+  OdsSelectCustomEvent,
+} from '@ovhcloud/ods-components';
+
+type SelectFieldProps<T extends Record<string, unknown>> = {
+  name: string;
+  label: string;
+  options: T[];
+  optionValueKey: keyof T;
+  optionLabelKey?: keyof T;
+  isDisabled: boolean;
+  isLoading?: boolean;
+  handleChange: (
+    event: OdsSelectCustomEvent<OdsSelectChangeEventDetail>,
+  ) => void;
+  placeholder?: string;
+  error?: string;
+};
+
+export const SelectField = <T extends Record<string, unknown>>({
+  name,
+  isDisabled,
+  isLoading,
+  handleChange,
+  options,
+  label,
+  optionLabelKey,
+  optionValueKey,
+  placeholder,
+  error,
+}: SelectFieldProps<T>) => {
+  return (
+    <OdsFormField error={error}>
```

Comment:
Maybe extends all props `OdsFormField` compatible with on this component ?

Reply:
Not sure of the level of control we'd like this component to have, I think we'll see with our usage if we have to provide `formFieldProps`, `selectProps`, `optionProps`, or `labelProps` in the future, or if it's better to add props step by step depending on our needs

Reply:
👍